test/fixtures/complete-response.js

```diff
@@ -0,0 +1,11 @@
+module.exports = function (bucket = 'bucket', key = 'test.txt') {
+  return `
+    <?xml version="1.0" encoding="UTF-8"?>
+    <CompleteMultipartUploadResult xmlns="http://s3.amazonaws.com/doc/2006-03-01/">
+      <Location>https://bucket.s3.amazonaws.com/${key}</Location>
```

Comment:
This bucket path requires `cloudfront` to be set, which is fine but we still need to spec the url when cloudfront is not set.